pkg/onit/cli/debug.go

```diff
@@ -53,18 +54,59 @@ func getDebugCommand() *cobra.Command {
 				exitError(err)
 			}
 
-			port, _ := cmd.Flags().GetInt("port")
-			if err := cluster.PortForward(args[0], port, 40000); err != nil {
+			nodes, err := cluster.GetNodes()
+			if err != nil {
 				exitError(err)
+			}
+
+			port, _ := cmd.Flags().GetInt("port")
+
+			if len(args) == 0 {
+				var wg sync.WaitGroup
+				n := len(nodes)
+				wg.Add(n)
+
+				asyncErrors := make(chan error, n)
+				freePorts, err := onit.GetFreePorts(n)
+				if err != nil {
+					exitError(err)
+				}
+
+				for index := range nodes {
+					go func(node onit.NodeInfo, port int) {
+						fmt.Println("Start Debugger for:", node.ID)
+						err = cluster.PortForward(node.ID, int(port), 40000)
+						asyncErrors <- err
+						wg.Done()
+					}(nodes[index], freePorts[index])
+
+				}
+
+				go func() {
+					wg.Wait()
+					close(asyncErrors)
+				}()
+
+				for err = range asyncErrors {
+					if err != nil {
+						exitError(err)
+					}
+				}
+
 			} else {
-				fmt.Println(port)
+				if err := cluster.PortForward(args[0], port, 40000); err != nil {
+					exitError(err)
+				} else {
+					fmt.Println(port)
+				}
+
 			}
 		},
 	}
 	cmd.Flags().StringP("cluster", "c", getDefaultCluster(), "the cluster for which to load the history")
```

Comment:
This flag help appears to be incorrect. Maybe a bunch of them are incorrect?

Reply:
I think that is the only one and will fix it. I will take a look at the other commands later.